e2e_tests/camino/cchain_nomock.test.ts

```diff
@@ -41,17 +67,32 @@ const kycAddr: string = "0x8db97c7cece249c2b98bdc0226cc4c2a57bf52fc"
 const gasFeeXAddr = "X-kopernikus13kyf72ftu4l77kss7xm0kshm0au29s48zjaygq"
 const gasFeeAddr: string = "0x305cea207112c0561033133f816d7a2233699f06"
 
+const ownerXAddr = "X-kopernikus1jla8ty5c9ud6lsj8s4re2dvzvfxpzrxdcrd8q7"
+const ownerPAddr = "P-kopernikus1jla8ty5c9ud6lsj8s4re2dvzvfxpzrxdcrd8q7"
+const ownerCAddr = "0x3187aFA01C28A200Bf27F605E2753E8d9CcBe0e1"
+
+const owner2PAddr = "P-kopernikus15hscuhlg5tkv4wwrujqgarne3tau83wrpp2d0d"
+const owner2CAddr = "0x60ac3Dd4A0f54db7bA0Ac913B55F55E4D42F83b1"
+
+const multiSigAliasPAddr = "P-kopernikus1fwrv3kj5jqntuucw67lzgu9a9tkqyczxgcvpst"
+const multiSigAliasCAddr = "C-kopernikus1fwrv3kj5jqntuucw67lzgu9a9tkqyczxgcvpst"
+
 const blacklistAddr: string = "0x7f28dcdfc67af590918c271226034058fd15e868"
 
 const dummyContractBin =
   "0x60806040523480156100115760006000fd5b50610017565b61016e806100266000396000f3fe60806040523480156100115760006000fd5b506004361061005c5760003560e01c806350f6fe3414610062578063aa8b1d301461006c578063b9b046f914610076578063d8b9839114610080578063e09fface1461008a5761005c565b60006000fd5b61006a610094565b005b6100746100ad565b005b61007e6100b5565b005b6100886100c2565b005b610092610135565b005b6000600090505b5b808060010191505061009b565b505b565b60006000fd5b565b600015156100bf57fe5b5b565b6040517f08c379a000000000000000000000000000000000000000000000000000000000815260040180806020018281038252600d8152602001807f72657665727420726561736f6e0000000000000000000000000000000000000081526020015060200191505060405180910390fd5b565b5b56fea2646970667358221220345bbcbb1a5ecf22b53a78eaebf95f8ee0eceff6d10d4b9643495084d2ec934a64736f6c63430006040033"
 
 let keystore: KeystoreAPI
 let contract: any
-let xChain, cChain: any
-let cAddressStrings: any
+let xChain, cChain, pChain: any
+let cAddresses, cAddressStrings: any
```

Comment:
Just curious why are all these vars any? I already asked nikos to clean up this definition section, it is really hard to overview...

Reply:
Thanks for your feedback. The reason that pChain, cChain, xChain are of type "any" is that some of their properties are used in the test cases (eg. blockchainID) and you can't really use them if they are strictly defined because these properties are protected.

For the rest of the vars there is not a particular reason to be "any". I fixed it.

Reply:
If you really need property access, I think it is not that bad to implement them in the TX itself.
Most properties are simply not implemented because there was no need before, but getters normally doesn't harm.